common/src/thrift/tachyon.thrift

```diff
@@ -27,8 +22,31 @@ struct ClientWorkerInfo {
   7: i64 starttimeMs
 }
 
-struct ClientFileInfo {
-  1: i32 id
+// Information about blocks.
```

Comment:
I think this comment should be moved to BlockInfo and instead we should say ``// Information about block location.`` here.